tri.css

```
@@ -66,94 +76,19 @@ h1{
   left: -302px;
 }
 
-.container1a{
+.container1a {
   display: flex;
-  left: 440px; 
+  left: 440px;
   line-height: 25px;
 }
 
-  .container1{
-    display: flex;
-    margin: 0 auto;
-    position: static;
-    line-height: 50px;
-  }
-  .container2{
-    display: flex;
-    margin: 0 auto;
-    line-height: 50px;
-  }
-  .container3{
-    display: flex;
-    margin: 0 auto;
-    line-height: 50px;
-  }
-  .container4{
-    display: flex;
-    margin: 0 auto;
-    line-height: 50px;
-  }
-  .container5{
-    display: flex;
-    position: relative;
-    margin: 0 auto;
-    line-height: 50px;
-  }
-
-.container1 .circle {
-  margin-right: auto;
-  margin-left: 60px;
-  height: 50px;
-  width: 50px;
-  border-radius: 50px;
-  text-align: center;
-  font-weight: bold;
-  font-size: 150%;
-  background-color: green;
-}
-
-.container2 .circle {
-  margin-right: auto;
-    margin-left: 60px;
-    height: 50px;
-    width: 50px;
-    display: block;
-    border-radius: 50px;
-    text-align: center;
-    font-weight: bold;
-    font-size: 150%;
-   background-color: green;
-}
-
-.container3 .circle {
-      margin-right: auto;
-      margin-left: 60px;
-      height: 50px;
-      width: 50px;
-      display: block;
-      border-radius: 50px;
-      text-align: center;
-      font-weight: bold;
-      font-size: 150%;
-      background-color: green;
-  float: left;
-}
-
-.container4 .circle {
-      margin-right: auto;
-      margin-left: 60px;
-      height: 50px;
-      width: 50px;
-      display: block;
-      border-radius: 50px;
-      text-align: center;
-      font-weight: bold;
-      font-size: 150%;
-      background-color: green;
-  float: left;
+.container1, .container2, .container3, .container4, .container5 {
+  display: flex;
+  margin: 0 auto;
+  line-height: 50px;
 }
 
-.container5 .circle {
+.container1 .circle, .container2 .circle, .container3 .circle, .container4 .circle, .container5 .circle {
```

You can minimize this by changing it to just the `.circle` selector
```suggestion
.circle {
```

---

Revised.  Thanks!